purchases/src/test/java/com/revenuecat/purchases/PurchasesCommonTest.kt

```diff
@@ -69,6 +71,10 @@ internal class PurchasesCommonTest: BasePurchasesTest() {
     private val mockLifecycle = mockk<Lifecycle>()
     private val mockLifecycleOwner = mockk<LifecycleOwner>()
 
+    @After
+    fun removeMocks() {
+        unmockkStatic(BillingClient::class)
+    }
```

Comment:
superclass `BasePurchasesTest` already has a `tearDown` method hence the naming here